test/cli/install/bun-upgrade.test.ts

```diff
@@ -176,3 +176,13 @@ it("zero arguments, should succeed", async () => {
   // Should not contain error message
   expect(stderr.toString()).not.toContain("error:");
 });
+
+import { escapePowershell } from "bun:internal-for-testing";
```

Comment:
Can we maybe move this to a new `test/unit` folder? Having these orthogonal tests glittered around behaviour tests isn't great.

Reply:
sure